app/assets/stylesheets/components/_header.scss

```diff
@@ -0,0 +1,17 @@
+.sc-TakeoverHeader {
+
+  &-content {
+    @include display(flex);
+    @include justify-content(space-between);
+  }
+
+  &--multipleCta {
+    @include display(flex);
```

Comment:
We don't need `display: flex;` here. Instead we should remove the divs wrapping the links.